README.md

```diff
@@ -70,10 +70,10 @@ Join us in creating the next generation of open trust infrastructure.
 
 ## Community and Next Steps 🎯
 
-We're currently working on a redesign of the application, including a revamp of the codebase so Documenso can be more intuitive to use and robust to develop upon.
+We're currently working on a redesign of the application, including a revamp of the codebase, so Documenso can be more intuitive to use and robust to develop upon.
```

Comment:
Does `We're currently redesigning the application, including a revamp of the codebase, so Documenso can be more intuitive and robust to use and develop upon.` sound better? What do you think?